config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main.yaml

```diff
@@ -48,7 +48,7 @@ periodics:
       # To check the available envtest in Kubebuilder, please
       # refer to https://github.com/kubernetes-sigs/kubebuilder/tree/tools-releases.
       - name: KUBEBUILDER_ENVTEST_KUBERNETES_VERSION
```

Comment:
Bumped mgmt cluster version of test-mink8s to v1.23 as new minimum mgmt cluster version is v1.23 now